hail_scripts/shared/elasticsearch_client.py

```diff
@@ -339,3 +339,31 @@ def set_index_meta(self, index_name, index_type_name, _meta):
         }
 
         self.es.indices.put_mapping(index=index_name, doc_type=index_type_name, body=index_mapping)
+
+    def route_index_to_temp_es_cluster(self, index_name, to_temp_loading):
```

Comment:
This function seems like it could be confusing since passing `to_temp_loading=False` makes it do the exact opposite of what its name suggests that it does. Maybe it could use a different name or be split into two functions?

Reply:
Agreed in theory, in practice this is used all over the place and I copy-pasted it exactly to make it part of a base class instead of a subclass so I'm really hesitant to actually make any changes to it